Makefile

```diff
@@ -39,12 +39,16 @@ ARFLAGS = rcs
 LDFLAGS = -pthread
 LDLIBS = $(TEST_LIBS)
 
+LN = ln
 CP = cp
 MKDIR = mkdir
 SUDO =
 
 CCTZ = cctz
 CCTZ_LIB = lib$(CCTZ).a
+CCTZ_SHARED_LIB = lib$(CCTZ).so.2.0
```

Comment:
I added a `CCTZ_VERSION_MAJOR` and `CCTZ_VERSION_MINOR` variable to make it a bit more obvious what those numbers are. Also allows those two variables to be overridden.